Description: Add webpacker to $LOAD_PATH in test_app
 test_app depends on webpacker, but could not load during build.
 .
 This patch will add webpacker ('../../lib') to $LOAD_PATH in test_app.
Author: Jongmin Kim <jmkim@pukyong.ac.kr>
Forwarded: not-needed
Last-Update: 2019-06-21
--- a/test/test_app/Rakefile
+++ b/test/test_app/Rakefile
@@ -1,3 +1,4 @@
+$:.unshift("../../lib")
 require_relative "config/application"
 
 Rails.application.load_tasks
